Sources/Publish/API/AnyDateFormatter.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AnyDateFormatter.swift
+//  
+//
+//  Created by Dustin Pfannenstiel on 3/10/20.
+//  Copyright (c) Dustin Pfannenstiel, 2020
+//  MIT license, see LICENSE file for details
```

Comment:
Please use the same start-of-file header as the rest of the project uses (but with your copyright, of course).